intermittent_task_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !docker
```

Comment:
You've disabled docker for lots of tests where the engine is not related to the test. This is one example.

Reply:
All tests that use the generic-worker payload will fail under docker builds. Since they run in other builds, I disabled them.